lib/LaTeXML/Package/expl3.lua.ltxml

```diff
@@ -0,0 +1,124 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  expl3.lua                                                          | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+# Translation of the intarray portion of expl3.lua
+# lua arrays are typically 1-based
+# Apparently an expl3 intarray is encoded at the TeX level by \__intarray:w <Number>
+# We store actual ints in the array, seems more efficient (could be Number or Tokens)
+DefParameterType('Intarray', sub {
+    my ($gullet) = @_;
+    my $tok = $gullet->readXToken;
+    if (my $defn = LookupDefinition($tok)) {    # token may be \let to the identifier!!!
+      $tok = $defn->getCS; }
+    if ($tok && $tok->equals(T_CS('\__intarray:w'))) {
+      my $n       = $gullet->readNumber;
+      my $address = '__intarray' . '_' . ToString($tok) . '_' . $n->valueOf;
```

Comment:
Not really a bug, but having every `$address` begin with `__intarray_\__intarray:w_` seems a bit redundant.  (Unless there's some other way to get an intarray that I wasn't finding.)